grid2op/Runner/Runner.py

```diff
@@ -479,19 +488,28 @@ def _new_env(self, chronics_handler, backend, parameters):
                             chronics_handler=chronics_handler,
                             backend=backend,
                             parameters=parameters,
+                            name=self.name_env,
                             names_chronics_to_backend=self.names_chronics_to_backend,
                             actionClass=self.actionClass,
                             observationClass=self.observationClass,
                             rewardClass=self.rewardClass,
                             legalActClass=self.legalActClass,
                             voltagecontrolerClass=self.voltageControlerClass,
                             other_rewards=self._other_rewards,
+
```

Comment:
indent

Reply:
I don't see indend issue on pycharm for windows :-/